uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/manage-leave-requests.html

```diff
@@ -19,7 +19,7 @@ <h3 class="col-md-5 panel-title" ng-if="!vm.isAdmin">Manager Absence Approval</h
             <button
               ng-repeat="filter in vm.filtersByAssignee"
               type="button"
-              ng-click="vm.refreshWithFilterByAssignee(filter.type)"
+              ng-click="vm.refreshWithFilterByAssignee(filter)"
```

Comment:
The function expects a `String`, but you are sending object. Please update the document of the function.